.editorconfig

```
@@ -1,10 +1,16 @@
 root = true
 
-[*.{adoc,bat,groovy,html,java,js,jsp,kt,kts,md,properties,py,rb,sh,sql,svg,txt,xml,xsd}]
+[*]
 charset = utf-8
+end_of_line = lf
+indent_style = space
```

is the intend space for all others as it was only specified on an explicit set?
```suggestion
indent_style = tab
```